FordLabs/PeopleMover
# 138 move tags to new sub nav menu

## Issue
Resolves 138

## What was done
- [x] Create new dropdown component
- [x] Create subnav section component
- [x] Consolidate disparate existing components into reusable components  
- [x] Created reusable tag management modal
- [x] Added new action for getting filters from the backend / local storage when space changes

## How to test
1. Open a space
2. Add `#newui` to the end of the URL
3. Explore the new filter dropdowns
4. Remove `#newui` from the URL
5. Verify the old UI still sorts and filters properly

## Accessiblity Criteria
### Testing

### Manual Checks
